apps/api/src/quantagent/api/observability/maintenance.py

```
@@ -0,0 +1,238 @@
+        if compressed_path.exists():
+            return MaintenanceSummary(skipped_files=1)
+
+        with parsed.path.open("rb") as source, gzip.open(compressed_path, "wb") as target:
+            shutil.copyfileobj(source, target)
+        parsed.path.unlink(missing_ok=True)
+        return MaintenanceSummary(compressed_files=1)
+
+
+def _compute_disk_guard_state(config: MaintenanceConfig) -> DiskGuardState:
+    root_dir = config.root_dir
+    # 请求路径会周期性查询 disk guard；未启用对应阈值时跳过昂贵的目录遍历和磁盘统计。
```

Ruff (0.15.14)

[warning] Comment contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF003)